src/System.Numerics.Vectors/ref/System.Numerics.Vectors.csproj

```diff
@@ -4,8 +4,10 @@
   <PropertyGroup>
     <AssemblyVersion>4.1.1.0</AssemblyVersion>
     <OutputType>Library</OutputType>
-    <PackageTargetFramework>dotnet5.4</PackageTargetFramework>
-    <NuGetTargetMoniker>.NETPlatform,Version=v5.4</NuGetTargetMoniker>
+    <PackageTargetFramework>dotnet5.2</PackageTargetFramework>
```

Comment:
For consistency we may want the ref folder for portable-net45+win8.  If you'd like to test it you can add a ValidationFramework entry to the pkgproj for portable-net45+win8.